source/sass/patterns/molecules/reference.scss

```diff
@@ -23,6 +23,7 @@
 }
 
 .reference__doi {
+  @include doi();
```

Comment:
When this is removed, the `letter-spacing: $font-letterspacing-label` still needs adding (to match eLife). Which shows the name is wrong (since this isn't a `label`).

Makes me wonder... are we _always_ applying this to small (`-3`, maybe more) text?

Reply:
This is the only bit I have not done as do not fully understand.

Reply:
This is an unrelated bug fix anyway, will wait for @davidcmoulton's return.